earthpy/spatial.py

```diff
@@ -168,6 +298,12 @@ def stack_raster_tifs(band_paths, out_path, arr_out=True):
         >>> # optionally, clean up example output
         >>> os.remove(destfile)
     """
+    
+    # Throw warning and exit
+    raise Warning("stack_raster_tifs is deprecated. Use stack(). Exiting...")
+    sys.exit()
+    
+    
     # Set default import to read
     kwds = {"mode": "r"}
```

Comment:
Do we need to retain the code below the warning above given we are deprecating this function @joemcglinchy

Reply:
keep it for now. when it is fully removed we can blow it away.